cli/dcoscli/data/help/package.txt

```diff
@@ -21,7 +21,7 @@ Usage:
     dcos package repo remove <repo-names>...
     dcos package search [<query> --json]
     dcos package uninstall <package-name>
-                           [--cli | [--app --app-id=<app-id> --all]]
+                           [--cli | [--app --app-id=<app-id> --all --yes]]
```

Comment:
Correct, the user will only need to confirm something when uninstalling an app.

They will specifically acknowledge _what_ app they are uninstalling